NOTICE.txt

```diff
@@ -1,6 +1,6 @@
 Mattermost Calls
 
-©2020-present Mattermost,Inc. All Rights Reserved. See LICENSE for license information.
+©2015-present Mattermost,Inc. All Rights Reserved. See LICENSE for license information.
```

Comment:
This would be inconsistent with the changes we made in https://github.com/mattermost/mattermost-plugin-calls/pull/952, which set 2020 as the starting year. We may need a small tweak to https://github.com/mattermost/notice-file-generator. /cc @lieut-data

Reply:
Thanks for the heads up, @streamer45! Yes, I ran into this limitation with the existing tool as well. I'll own next steps to update the tooling. For now, propose we leave this PR unmerged but still open to avoid the tool just recreating again.

Reply:
Turns out the configuration is right in the repository -- pushed a PR here to update same.